containers-toolkit/containers-toolkit.psd1

```diff
@@ -18,11 +18,13 @@
     Copyright         = '(c) Microsoft Corporation. All rights reserved.'
 
     # Description of the functionality provided by this module
-    Description       = 'PowerShell functions that allow you to download, install, and configure Containerd, Buildkit, Nerdctl, and Windows CNI plugins'
+    Description       = 'PowerShell functions that allow you to download, install, and configure Containerd, Buildkit, nerdctl, and Windows CNI plugins'
 
     # Minimum version of the PowerShell engine required by this module
     PowerShellVersion = '5.0'
 
+    RequiredModules   = @('HNS', 'ThreadJob')
```

Comment:
Is HNS available by default or is it enabled when you enable Containers Feature?

Reply:
It is available in some devices. In a WS2019, had to install the module, but not the case for WS22.